tree/tree/test/CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 
 ROOT_STANDARD_LIBRARY_PACKAGE(ElementStruct NO_INSTALL_HEADERS HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/ElementStruct.h SOURCES ElementStruct.cxx LINKDEF ElementStructLinkDef.h DEPENDENCIES RIO)
+if(MSVC)
+  add_custom_command(TARGET ElementStruct POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>/libElementStruct.dll
+                                     ${CMAKE_CURRENT_BINARY_DIR}/libElementStruct.dll)
+endif()
```

Comment:
How about
```cmake
if(MSVC)
  link_directories(${CMAKE_CFG_INTDIR})
endif()
```
neat the top of the file? This would probably solve the problem if other libraries need to be linked as well.

Reply:
Well, the DLL is needed when running a test, not when linking, but I could add it in the PATH.... Will try